src/main/java/org/jahia/modules/sitemap/utils/VanityUrls.java

```diff
@@ -55,6 +56,10 @@ public static String getActiveUrl(JCRNodeWrapper node, String langCode) throws R
         return getActiveUrl(vanityUrls);
     }
 
+    public static boolean canUseVanity() {
```

Comment:
we should probably rename as this is unrelated to vanity settings. I suggest maybe `urlRewriteEnabled()` or along those lines instead. And since this is unrelated to Vanity URLs also would make sense  to move it probably to the more generic Utils class instead.

Reply:
That makes sense.